core/frontend/src/ViewCreator2d.ts

```diff
@@ -20,7 +20,8 @@ import { IModelConnection } from "./IModelConnection";
 import { ViewState, ViewState2d } from "./ViewState";
 import { DrawingViewState } from "./DrawingViewState";
 import { SheetViewState } from "./SheetViewState";
-import { EntityState, loggerCategory } from "./imodeljs-frontend";
+import { EntityState } from "./EntityState";
+import { loggerCategory } from "./extension/Extension";

```

Comment:
I removed the import of `loggerCategory` - it didn't make sense for ViewCreator to log using the "extension" category, and it doesn't really need to log at all.